examples/get-access-report.py

```diff
@@ -93,7 +93,10 @@ def get_connection_data(user_id):
             connections_data["user"] = connection_data["user"]["display_name"]
             for p in data_list:
                 if p in ("connected", "disconnected"):
-                    connection_data[p] = connection_data[p].split(".")[0]
+                    if connection_data["status"] == "CONNECTED" and p == "disconnected":
+                        connection_data[p] = "CONNECTED"
+                    else:
+                        connection_data[p] = connection_data[p].split(".")[0]
```

Comment:
Our current Cyclomatic complexity checker complains about too many branching (loopings, conditionals) in this function. The checking generally means that it is difficult to sufficiently test the code. Another complexity checker, the Cognitive complexity checker, will be stricter and would penalize nestings of loops and conditionals. This checking generally means that it is difficult to understand the function while reading it.

Both checks would hint that the function is doing more than it should. In this case, you can have a function to return all raw connection data from PrivX and another to format it into desired CSV structure before writing to disk.